package.json

```diff
@@ -150,7 +150,8 @@
 		},
 		"@mikro-orm/migrations-mongodb": {
 			"mongodb": "^4.17.0"
-		}
+		},
+		"validator": "^13.15.22"
```

Comment:
why not using the newest version 13.15.23 ?
also do we need to leave some kind of comment somewhere to say "when" we can remove that override again?